.travis.yml

```diff
@@ -1,6 +1,21 @@
-sudo: required
+sudo: false
+dist: trusty
 language: node_js
 node_js:
-  - 8
+  - 'lts/*'
+before_install:
+- npm install -g greenkeeper-lockfile@1
+before_script:
+  - export CHROME_BIN=/usr/bin/google-chrome
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
```

Comment:
Is this still necessary or does the vjs karma config run on normal Chrome and not headless by default?

Reply:
Not sure, let me check